http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreterSpec.scala

```diff
@@ -141,6 +141,10 @@ class AKSInterpreterSpec extends AnyFlatSpecLike with TestComponent with Leonard
       "identity.resourceId=identity-id," +
       "identity.clientId=identity-client-id," +
       "sam.url=https://sam.dsde-dev.broadinstitute.org/," +
+      "cbas.coaEnabled=true," +
```

Comment:
Just double-checking here (e.g. if we need to set some of these false) -- I assume we are OK with all being `enabled` from the Leo side, then overriding those values in the downstream Helm chart?

Reply:
sounds good!